### PR TITLE
Create 2023-11-10-Poloniex looses $125 million in heist

### PR DESCRIPTION
**Pull request by aholman0725**

### CODE REVIEW

File: content/attacks/posts/2023-11-10-PoloniexHeist [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+date: 2023-11-10
+target-entities: Poloniex
+entity-types:
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-PoloniexHeist:4]: see issue for stnadard entities

File: content/attacks/posts/2023-11-10-PoloniexHeist [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Attackers
+
+The identity of the attacker is currently unknown.
+
+## Losses
+
+-Poloniex [lost](https://twitter.com/lookonchain/status/1722969227915661531) 56,000,000 worth of top altcoin ETH, 48,000,000 worth of [TRX,](https://platform.arkhamintelligence.com/explorer/address/TXs71PNcHmxgptA3wQ4g398qbToMPhnEoG) and 18,000,000 worth of BTC.
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-PoloniexHeist:31]: delete: "top altcoin"

File: content/attacks/posts/2023-11-10-PoloniexHeist [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+-The attacker was also able to obtain a number of other crypto assets, including stablecoins Tether (USDT), USD Coin (USDC), as well as popular [memecoin](https://cryptopotato.com/heres-what-tokens-were-stolen-in-the-125-million-poloniex-hack/) Shiba Inu (SHIB).
+
+## Timeline
+
+November 10, 2023: Poloniex announces their wallet had been closed for [maintenance](https://twitter.com/PoloSupport/status/1722934043543752968?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1722934043543752968%7Ctwgr%5E58cb76ede3fa88c84e8020c747d8943bf3d02bdc%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.fxstreet.com%2Fcryptocurrencies%2Fnews%2Fcrypto-exchange-poloniex-hack-leads-to-60-million-in-assets-stolen-peckshield-says-202311101206) before announing the hack and the extent of the losses just an [hour](https://www.fxstreet.com/cryptocurrencies/news/crypto-exchange-poloniex-hack-leads-to-60-million-in-assets-stolen-peckshield-says-202311101206) later.
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-PoloniexHeist:37]: timeline should be a sequential list
date/tiome (UTC if known): wallet closed annonced
date/time: announced extent of losses

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by aholman0725** [comment]
Requested changes completed.

**Comment by aholman0725** [comment]
What's the process for updating an pull request once more information is released?

File: content/attacks/posts/2023-11-10-PoloniexHeist (hunk 3 above)
Comment (jhirschkorn) [reply content/attacks/posts/2023-11-10-PoloniexHeist]: Screenshot 2023-11-20 at 10 06 33 AM

Comment (aholman0725) [reply content/attacks/posts/2023-11-10-PoloniexHeist]: > Screenshot 2023-11-20 at 10 06 33 AM

Timeline format updated.

File: content/attacks/posts/2023-11-10-PoloniexHeist [hunk 4]
```diff
@@ -24,9 +24,9 @@ The identity of the attacker is currently unknown.
...
-November 10, 2023/12:06: Poloniex announces the hack and the extent of the [losses.](https://www.fxstreet.com/cryptocurrencies/news/crypto-exchange-poloniex-hack-leads-to-60-million-in-assets-stolen-peckshield-says-202311101206)
-November 15, 2023/12:30: Poloniex puts out a statement that the platform is now running smoothly and the [restoration](https://support.poloniex.com/hc/en-us/articles/19080588734743) efforts have almost been completed.
+**November 10, 2023, 11:07 AM UTC:** Poloniex announces their wallet had been closed for [maintenance](https://twitter.com/PoloSupport/status/1722934043543752968?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1722934043543752968%7Ctwgr%5E58cb76ede3fa88c84e8020c747d8943bf3d02bdc%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.fxstreet.com%2Fcryptocurrencies%2Fnews%2Fcrypto-exchange-poloniex-hack-leads-to-60-million-in-assets-stolen-peckshield-says-202311101206) 
+**November 10, 2023, 12:06 PM:** Poloniex announces the hack and the extent of the [losses.](https://www.fxstreet.com/cryptocurrencies/news/crypto-exchange-poloniex-hack-leads-to-60-million-in-assets-stolen-peckshield-says-202311101206)
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-PoloniexHeist:28]: add "UTC" to the times

**Review by jhirschkorn — APPROVED** [review]

**Comment by evgenydmitriev** [comment]
@aholman0725 fix the name of the file (look at other articles) and information about alleged attackers.

**Comment by aholman0725** [comment]
Yes ma’am. I knew we were forgetting something.  With warm regards,~Amanda Amanda O’ConnorVirtual Administrative AssistantShineBright Freelancing(516) 308-1432 From: AlbinaSent: Wednesday, November 22, 2023 4:01 AMTo: 1712n/dn-instituteCc: aholman0725; MentionSubject: Re: [1712n/dn-institute] 2023-11-10-Poloniex (PR ***@***.*** could you email your BTC address to ***@***.*** ? Please include the QT code.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Pull request merged**